apps/masterbots.ai/lib/hooks/use-thread.tsx

```
@@ -113,85 +118,85 @@ export function ThreadProvider({ children }: ThreadProviderProps) {
     },
     async onFinish(message: AIMessage) {
       await saveNewMessage({
-        role: 'assistant',
+        role: "assistant",
         threadId: activeThread?.threadId,
         content: message.content,
-        jwt: session!.user?.hasuraJwt
-      })
-    }
-  })
+        jwt: session!.user?.hasuraJwt,
```

Biome

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)